publish.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -xe
+
+# Install local teads-central
+docker run --rm -i docker-registry.teads.net/curl:master -sL http://ebzpub.ebuzzing.com/files/teads-central/get.sh | sh - 
+
+IMAGE=$(./teads-central vars image)
+HASH=$(./teads-central vars hash)
+VERSION=$(docker run $IMAGE:$HASH version | tail -1 | sed s/" "//g | sed s/v//g)
+
+docker build -t ${IMAGE}:9.6-alpine 9.6/alpine
```

Comment:
I would go for dropping the alpine suffix in our own registry to be consistent with other images, just using version as tag